src/EA.Weee.Web/Areas/Aatf/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -73,6 +74,9 @@ public class ManageEvidenceNotesController : AatfEvidenceBaseController
 
             using (var client = this.apiClient())
```

Comment:
@GuidoCangelosi Looks like the unit tests in ManageEvidenceNotesControllerIndexTests need updating to ensure that the GetAatfSummaryRequest and GetAatfNotesRequest requests are called with the expected compliance year. This is examples of this in the Sscheme Manage evidence note controller tests